mariamihaelafainarea/assignment1/SinglyLinkedList

```diff
@@ -0,0 +1,194 @@
+import java.util.Scanner;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author maria
+ */
+public class SinglyLinkedList<T> {
```

Comment:
If I have a list of 
"a"
"b"
"c"

I would expect the 0-th from the end element to be "c", 1-st from the end to be "b" and 2nd to be "a". Your algorithm works a little differently right now. What does it return for those 3 cases?

Reply:
I have done it. The problem was a bad index